src/sign-verify/clu_x509_verify.c

```diff
@@ -0,0 +1,179 @@
+/* clu_x509_verify.c
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#include <wolfclu/clu_header_main.h>
+#include <wolfclu/clu_log.h>
+#include <wolfclu/clu_optargs.h>
+#include <wolfclu/sign-verify/clu_verify.h>
+#include <wolfclu/x509/clu_cert.h>
+
+static struct option verify_options[] = {
+    {"CAfile",    required_argument, 0, WOLFCLU_INFILE    },
+    {"crl_check", no_argument,       0, WOLFCLU_CHECK_CRL },
+    {"help",      no_argument,       0, WOLFCLU_HELP      },
+    {"h",         no_argument,       0, WOLFCLU_HELP      },
+
+    {0, 0, 0, 0} /* terminal element */
+};
+
+
+static void wolfCLU_x509VerifyHelp(void)
+{
+    WOLFCLU_LOG(WOLFCLU_L0, "./wolfssl verify -CAfile <ca file name> "
+            "[-crl_check] <cert to verify>");
+}
+
+
+int wolfCLU_x509Verify(int argc, char** argv)
+{
+    int ret    = WOLFCLU_SUCCESS;
+    int inForm = PEM_FORM;
+    int crlCheck  = 0;
```

Comment:
```
src/sign-verify/clu_x509_verify.c:49:9: error: variable ‘crlCheck’ set but not used [-Werror=unused-but-set-variable]

   49 |     int crlCheck  = 0;

      |         ^~~~~~~~
```

Reply:
I used the wolfSSL `./configure --enable-wolfclu`. Looks like `HAVE_CRL` is missing.

Reply:
should work without HAVE_CRL too, will fix the warning